packages/rpc-subscriptions-api/src/index.ts

```diff
@@ -1,3 +1,40 @@
+/**
+ * This package contains types that describe the [methods](https://solana.com/docs/rpc/websocket) of
+ * the Solana JSON RPC Subscriptions API, and utilities for creating a
+ * {@link RpcSubscriptionsApi} implementation with sensible defaults. It can be used standalone, but
+ * it is also exported as part of Kit
+ * [`@solana/kit`](https://github.com/anza-xyz/kit/tree/main/packages/kit).
+ *
+ * Each RPC subscriptions method is described in terms of a TypeScript type of the following form:
+ *
+ * ```ts
+ * type ExampleApi = {
+ *     thingNotifications(address: Address): Thing;
+ * };
+ * ```
+ *
+ * A {@link RpcSubscriptionsApi} that implements `ExampleApi` will ultimately expose its defined
+ * methods on any {@link RpcSubscriptions} that uses it.
+ *
+ * ```ts
+ * const rpcSubscriptions: RpcSubscriptions<ExampleApi> = createExampleRpcSubscriptions(/* ... *\/);
+ * const thingNotifications = await rpc
+ *     .thingNotifications(address('95DpK3y3GF7U8s1k4EvZ7xqyeCkhsHeZaE97iZpHUGMN'))
+ *     .subscribe({ abortSignal: AbortSignal.timeout(5_000) });
+ * try {
+ *     for await (const thing of thingNotifications) {
+ *         console.log('Got a thing', thing);
+ *     }
+ * } catch (e) {
+ *     console.error('Our subscription to `Thing` notifications has failed', e);
+ * } finally {
+ *     console.log('We are done listening for `Thing` notifications');
+ * }
+ * ```
+ *
+ * @pacakageDocumentation
```

Comment:
```suggestion
 * @packageDocumentation
```

Reply:
Knew I was going to do this one of these days.